staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -134,6 +134,8 @@ type Config struct {
 	DiscoveryAddresses discovery.Addresses
 	// The default set of healthz checks. There might be more added via AddHealthzChecks dynamically.
 	HealthzChecks []healthz.HealthzChecker
+	// The default set of readyz-only checks. There might be more added via AddReadyzChecks dynamically.
```

Comment:
this is the config struct, not really the default, but the actual set of checks.

Reply:
ah, this was a stray comment from an iteration long ago.

Reply:
It turns out I actually stole this comment from you. You wrote the initial comment two lines above for HealthzChecks three years ago.

Reply:
Since that time it was moved around 7 times probably. Never trust existing code or comments ;-)

Reply:
It actually does make sense. We do assign the default checks to this field. And additional checks are, in fact, added dynamically.